concepts/05 UI Components/TagBox/00 Getting Started with TagBox/50 Group Data.md

```diff
@@ -0,0 +1,9 @@
+#include tutorials-editors-group-data
+
+There are additional search properties demonstrated in the following demo:
```

Comment:
Unnecessary "there are".
```suggestion
Additional search properties demonstrated in the following demo:
```

Reply:
The same point as above: 

But then it is not a valid sentence. It still requires passive: 

'Additional search properties are demonstrated in the following demo:'

However, I would prefer the following: 'This demo shows additional search properties: '

Reply:
Yes, you are correct. As before, I like your version.

```suggestion
This demo shows additional search properties:
```